Makefile

```
@@ -416,75 +430,40 @@ define _run_for_selected_projects
-	@$(PROJECT_FLEXT_INFRA) workspace serialize-make --workspace "$(PROJECT_ROOT)" --makefile "$(SELF_MAKEFILE)" --verb "docs" --selector-value "$(WHAT)" --apply-token "$(APPLY)"
-
-_serialized_docs:
-	$(call _dispatch,docs)
+help:
+	$(call _dispatch,$@)
 
 
+# `all` = clean setup gen fmt fmt fix check test. CI=Y skips pytest inside
+# make test (flext_infra._pytest_entry guards mro-v4p5), so all always calls
+# test — the pytest gate is self-guarding.
+all: _builtin_require_environment
```

checkmake (0.3.2)

[warning] Target body for "all" exceeds allowed length of 5 lines (11).

(maxbodylength)